src/redux/data.ts

```diff
@@ -57,17 +57,37 @@ export interface IncomeW2 {
 }
 
 export enum Income1099Type {
-  INT = 'INT'
+  B = 'B',
+  INT = 'INT',
+  DIV = 'DIV'
 }
 
 export const form1099Types: Income1099Type[] = [
-  Income1099Type.INT
+  Income1099Type.B,
+  Income1099Type.INT,
```

Comment:
What is the advantage of having the 1099 form type strings in an enum vs putting them directly in the form1099Types array directly like

form1099Types: string[] = [
  'B',
  'INT',
  'DIV'
] 

or perhaps using the enum directly?

Reply:
Given an enum I don't think you can iterate over its keys without defining an array like this, is that not right?

Reply:
I think that you can get the values using `Object.values(MyEnum)` but that might decrease clarity

https://stackoverflow.com/questions/39372804/how-can-i-loop-through-enum-values-for-display-in-radio-buttons

Reply:
Ah Object.values does work, doesn't need an explicit type cast. Good!